README.md

```diff
@@ -22,43 +26,44 @@ Azure IoT Edge extension <sup>Preview</sup> makes it easy to code, build, deploy
 - It's also recommended to install [Docker Support for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=PeterJausovec.vscode-docker) to manage your IoT Edge Docker images, containers and registries. 
 
 For C# developers, you can develop, debug and deploy [C# modules](https://docs.microsoft.com/azure/iot-edge/tutorial-csharp-module) and [C# Functions on IoT Edge](https://docs.microsoft.com/azure/iot-edge/tutorial-deploy-function)
-- [IoT Edge C# module template](https://www.nuget.org/packages/Microsoft.Azure.IoT.Edge.Module)
-  ```
-  dotnet new -i Microsoft.Azure.IoT.Edge.Module
-  ```
-- [IoT Edge C# function template](https://www.nuget.org/packages/Microsoft.Azure.IoT.Edge.Function)
-  ```
-  dotnet new -i Microsoft.Azure.IoT.Edge.Function
-  ```
 - [.Net Core 2.0 SDK](https://www.microsoft.com/net/download/core)
 - [C# for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
 
+For Python developers, you can develop and deploy [Python module](https://github.com/Azure/cookiecutter-azure-iot-edge-module)
```

Comment:
I prefer directing Python module developers to documentation on Docs, instead of the cookiecutter template GitHub.

Reply:
docs.microsoft.com/azure/iot-edge/tutorial-python-module